getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>"Vector point at north pole is..."</p><p>This seems ungrammatical. Maybe "The vector pointing to the north pole represents"? Or simply "The north pole represents"?</p><p> </p><p>It should also be worth mentioning that quantum states are represented by points <em>on</em> the Bloch sphere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>